dspace-server-webapp/src/test/java/org/dspace/app/rest/EPersonRestRepositoryIT.java

```diff
@@ -3129,4 +3130,199 @@ public void findByMetadataUsingFirstNamePaginationTest() throws Exception {
 
     }
 
-}
+    @Test
+    public void validatePasswordRobustnessContainingAtLeastAnUpperCaseCharUnprocessableTest() throws Exception {
+        configurationService.addPropertyValue("eperson.password.regex-validation.pattern", "^(?=.*[A-Z])");
```

Comment:
Shouldn't this be `setProperty()` instead of `addPropertyValue()`?  The `addPropertyValue()` method turns this setting into an array of values, instead of a single value.  In this case it appears to not matter since the default value is empty... but I'd recommend changing this to `setProperty()` as it should never be an array.

This same comment goes for all the new tests in this IT.  They all are wrongly using `addPropertyValue()`